aiodns/__init__.py

```diff
@@ -75,21 +75,21 @@ def query(self, host, qtype):
             qtype = query_type_map[qtype]
         except KeyError:
             raise ValueError('invalid query type: {}'.format(qtype))
-        fut = asyncio.Future(loop=self.loop)
+        fut = asyncio.Future(loop=self.loop)  # type: asyncio.Future
```

Comment:
Actually _mypy_ with the `strict`  option reports `Missing type parameters for generic type "Future"`. Indeed, nobody knows what will be returned when that future will be awaited.

Reply:
@cooperlees Sorry. Looks like futures can't be annotated.